docs/run-platform-server/ingestion-filtering.mdx

```diff
@@ -3,85 +3,46 @@ title: Ingestion Filtering
 order: 46
 ---
 
-The Ingestion Filtering feature is now released for public beta testing available from Horizon [version 2.18.0](https://github.com/stellar/go/releases/tag/horizon-v2.18.0) and up.
-
 ## Overview
 
-Ingestion Filtering enables Horizon operators to drastically reduce storage footprint of their Horizon DB by whitelisting Assets and/or Accounts that are relevant to their operations. This feature is ideally suited for private Horizon operators who do not need full history for all assets and accounts on the Stellar network.
+Ingestion Filtering enables Horizon operators to drastically reduce storage footprint of the historical data on Horizon DB by whitelisting Assets and/or Accounts that are relevant to their operations.
```

Comment:
I know It it is already in our existing doc but consider changing it to  
more industry preferred term like "allowlisting".

Reply:
+1, it shows up in a few other places in here as well